src/cmd/tools/query_index_segments/main/main.go

```diff
@@ -189,11 +190,16 @@ func run(opts runOptions) {
 				log.Fatal("search execute error", zap.Error(err))
 			}
 
+			reader := docs.NewEncodedDocumentReader()
 			fields := make(map[string]string)
 			for iter.Next() {
 				d := iter.Current()
+				m, err := docs.GetFromDocument(d, reader)
```

Comment:
Can we rename `GetFromDocument` to `MetadataFromDocument`?

Reply:
Fixed!